src/subscription/UpgradeConfirmPage.ts

```diff
@@ -158,7 +156,7 @@ export class UpgradeConfirmPage implements WizardPageN<UpgradeSubscriptionData>
 		return [
 			m(".flex-space-around.flex-wrap", [
 				m(".flex-grow-shrink-half.plr-l.flex-center.items-end",
-					m("img[src=" + HabReminderImage + "].pt.bg-white.border-radius", {
+					m("img[src=" + VisSignupImage + "].pt.bg-white.border-radius", {
```

Comment:
could you please use `src` like in the other place? Just because we should have done it before.